Index: flightreservation/src/main/java/edu/miu/cs/flightreservation/repository/personRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package edu.miu.cs.flightreservation.repository;\n\nimport edu.miu.cs.flightreservation.model.Person;\nimport org.springframework.data.jpa.repository.JpaRepository;\n\npublic interface personRepository extends JpaRepository<Person,Long> {\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/flightreservation/src/main/java/edu/miu/cs/flightreservation/repository/personRepository.java b/flightreservation/src/main/java/edu/miu/cs/flightreservation/repository/personRepository.java
--- a/flightreservation/src/main/java/edu/miu/cs/flightreservation/repository/personRepository.java	(revision bcb1ef090aa915972371831f489e34b7a57368f1)
+++ b/flightreservation/src/main/java/edu/miu/cs/flightreservation/repository/personRepository.java	(date 1628522263813)
@@ -2,7 +2,9 @@
 
 import edu.miu.cs.flightreservation.model.Person;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
-public interface personRepository extends JpaRepository<Person,Long> {
-
-}
+@Repository
+public interface PersonRepository extends JpaRepository<Person,Long> {
+   Person findPersonByUsername(String username);
+}
\ No newline at end of file
